dao/mongo_helper.go

```diff
@@ -58,19 +58,19 @@ func getInsolvencyPractitionersDetails(practitionersString string, transactionID
 
 func getPractitioners(practitionerIDs []string, collection *mongo.Collection) ([]models.PractitionerResourceDao, error) {
 	var practitionerResourceDaos []models.PractitionerResourceDao
-	var practitionerResourceDao models.PractitionerResourceDao
 
 	matchQuery := bson.D{{"$match", bson.D{{"data.practitioner_id", bson.D{{"$in", practitionerIDs}}}}}}
 	lookupQuery := bson.D{{"$lookup", bson.D{{"from", AppointmentCollectionName}, {"localField", "data.practitioner_id"}, {"foreignField", "practitioner_id"}, {"as", "data.appointment"}}}}
 	unwindQuery := bson.D{{"$unwind", bson.D{{"path", "$data.appointment"}, {"preserveNullAndEmptyArrays", true}}}}
 
 	// Retrieve practitioners and appointments from DB
-	practitionerCursor, err := collection.Aggregate(context.Background(), mongo.Pipeline{lookupQuery, matchQuery, unwindQuery})
+	practitionerCursor, err := collection.Aggregate(context.Background(), mongo.Pipeline{matchQuery, lookupQuery, unwindQuery}) //, unwindQuery, projectQuery})
```

Comment:
Minor tidying: The comment here could be removed for clarity.

Reply:
Fixed .Thanks